src/components/atoms/List.svelte

```diff
@@ -2,18 +2,45 @@
   // Props
   export let href
   export let isClickable = false
+  export let isNav = false
+  let active = true
 </script>
 
 <style lang="scss">
-  a,
-  a:visited {
-    color: #FFFFFF;
+  @import 'src/styles/index.scss';
+  li {
+    width: 100%;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: center;
+    &.isNav {
+      height: 50px;
+      a {
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+        align-items: center;
+        width: 100%;
+        height: 100%;
+      }
+    }
+    &.active {
+      font-family: $bold-font;
+      border-bottom: 1px solid $ui-blue;
+    }
+  }
+  a {
+    color: $ui-black;
     text-decoration: none;
+    // &:visited {
```

Comment:
Klopt het dat er nog comments in de code horen te zitten?

Reply:
Yess, this is for the slider (today, yesterday etc.) that still needs to be adapted for mobile and desktop! It has nothing to do with the navigation in this case :)